package.json

```diff
@@ -1,15 +1,18 @@
 {
   "devDependencies": {
     "grunt": "^0.4.5",
+    "grunt-autoprefixer": "^3.0.0",
     "grunt-aws-s3": "^0.9.4",
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-copy": "^0.7.0",
+    "grunt-contrib-cssmin": "^0.12.3",
```

Comment:
All the cool kids are using [clean-css](https://github.com/jakubpawlowicz/clean-css) these days.